digid_eherkenning/models/certificates.py

```diff
@@ -134,6 +139,16 @@ class ConfigCertificate(models.Model):
             "matching candidate will automatically be selected by the configuration."
         ),
     )
+    activation_date = models.DateTimeField(
+        # Ofset the time slightly to prevent it from possibly coming
+        # before certificate.valid_from
+        default=timezone.now() + timedelta(minutes=5),
```

Comment:
this evaluates to a timestamp when the module is imported - you can see that a fixed datetime is recorded in your migration.

I wouldn't even go with a default here to be honest - you can make the field nullable and if no value is provide look at the certificate start date. If a datetime is specified here, then use it.